src/main/java/com/minecolonies/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -46,51 +49,49 @@ public void onDestroyed()
     {
         if (hasWorker())
         {
-            //  EntityCitizen will detect the workplace is gone and fix up it's Entity properly
+            // EntityCitizen will detect the workplace is gone and fix up it's
+            // Entity properly
             removeCitizen(worker);
         }
 
         super.onDestroyed();
     }
 
     /**
-     * Returns the worker of the current building
+     * Returns the worker of the current building.
      *
-     * @return          {@link CitizenData} of the current building
+     * @return {@link CitizenData} of the current building
      */
     public CitizenData getWorker()
     {
         return worker;
     }
 
     /**
-     * Returns whether or not the building has a worker
+     * Returns whether or not the building has a worker.
      *
-     * @return          true if building has worker, otherwise false.
+     * @return true if building has worker, otherwise false.
      */
     public boolean hasWorker()
     {
         return worker != null;
     }
 
     /**
-     * Returns the {@link net.minecraft.entity.Entity} of the worker
+     * Returns the {@link net.minecraft.entity.Entity} of the worker.
      *
-     * @return          {@link net.minecraft.entity.Entity} of the worker
+     * @return {@link net.minecraft.entity.Entity} of the worker
      */
-    public EntityCitizen getWorkerEntity()
-    {
-        return (worker != null) ? worker.getCitizenEntity() : null;
-    }
+    public EntityCitizen getWorkerEntity(){ return (worker == null) ? null : worker.getCitizenEntity(); }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)